graylog2-server/src/test/java/org/graylog2/indexer/searches/SearchesIT.java

```diff
@@ -573,7 +573,7 @@ public void scrollReturnsMultipleChunksRespectingLimit() throws Exception {
 
         when(indexSetRegistry.getForIndices(Collections.singleton("graylog_0"))).thenReturn(Collections.singleton(indexSet));
         final AbsoluteRange range = AbsoluteRange.create(new DateTime(2015, 1, 1, 0, 0, DateTimeZone.UTC).withZone(UTC), new DateTime(2015, 1, 2, 0, 0, DateTimeZone.UTC).withZone(UTC));
-        final ChunkedResult scrollResult = searches.scroll("*", range, 5, 0, Collections.singletonList("source"), null, 2);
+        final ChunkedResult scrollResult = searches.scroll("*", range, 5, 0, Collections.singletonList("source"), null, Set.of(), 2);
```

Comment:
With new method param for `streams`, couldn't you add at least one test that actually tests `scroll()` with `streams` param different than `Set.of()` ?